tests/unit/models/reddit/test_live.py

```diff
@@ -0,0 +1,64 @@
+import pickle
+
+import pytest
+from praw.models import LiveThread
+
+from ... import UnitTest
+
+
+class TestLiveThread(UnitTest):
+
+    def test_construct_failure(self):
+        message = 'Either `id` or `_data` must be provided.'
+        with pytest.raises(TypeError) as excinfo:
+            LiveThread(self.reddit)
+        assert str(excinfo.value) == message
+
+        with pytest.raises(TypeError) as excinfo:
+            LiveThread(self.reddit, id='dummy', _data={'id': 'dummy'})
+        assert str(excinfo.value) == message
+
+    def test_equality(self):
+        thread1 = LiveThread(self.reddit, id='dummy1')
+        thread2 = LiveThread(self.reddit, id='Dummy1')
+        thread3 = LiveThread(self.reddit, id='dummy3')
+        assert thread1 == thread1
+        assert thread2 == thread2
+        assert thread3 == thread3
+        assert thread1 != thread2  # live thread ID in a URL is case sensitive
```

Comment:
⭐️ great realization that live thread ids are case sensitive. I definitely would have missed that.